massa-execution/src/sce_ledger.rs

```diff
@@ -32,7 +32,9 @@ impl SCELedgerEntry {
         }
 
         // module
-        self.opt_module = update.update_opt_module.clone();
+        if let Some(opt_module) = &update.update_opt_module {
+            self.opt_module = opt_module.clone();
+        }
```

Comment:
Can just be 
```rust
        if let Some(module) = update.update_opt_module {
            self.opt_module = Some(module);
        }
```

without manage all the process an Option right? You never make a real use of the super Option